wordpress-mobile/WordPress-iOS
# Remove large titles from the Reader and Notification tab

Fixes #20271

This PR removes the large titles from the Reader and Notification tab.

| Reader | Notifications |
| ------- | ------------- |
| ![](https://user-images.githubusercontent.com/9609223/223388043-8e053d05-db3f-4017-aa8f-abe211d037f1.png) | ![](https://user-images.githubusercontent.com/9609223/223387929-081dac0b-3b29-42be-a945-d43793988951.png) |

## Test Instructions
1. Install WordPress / Jetpack app.
2. Head to **Reader** tab.
3. **Verify** the navigation bar title is not large.
4. Repeat 2-3 steps for **Notifications** tab.

## Regression Notes
1. Potential unintended areas of impact
None.

6. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual Tests.

7. What automated tests I added (or what prevented me from doing so)
None.

## PR submission checklist:
